src/runtime/opengl.cpp

```diff
@@ -14,110 +14,112 @@
 
 // List of all OpenGL functions used by the runtime. The list is used to
 // declare and initialize the dispatch table in OpenGLState below.
-#define USED_GL_FUNCTIONS                                               \
```

Comment:
yes, but it still makes reviewing unnecessary painful. please undo that commit for review purposes.